roles/ngi-methylseq/templates/ngi-methylseq_site.config

```diff
@@ -1 +1,2 @@
 params.project = '{{ item.project_id }}'
+process.container = '{{ ngi_resources }}/singularity/ngi-methylseq/ngi-methylseq-{{ ngi_methylseq_version }}.img'
```

Comment:
Arguably add 
`ngi_methylseq_docker: "docker://scilifelab/ngi-methylseq"`

Reply:
Why?

Reply:
Refferred to wrong document because I'm a pleb. I mean to default/main.yml

Reply:
👍